### DISCUSSION TITLE
Triaging consensus levels

### DISCUSSION THREAD

**Discussion by stephen-rowan** [NFT-DAO/Governance-HOLON#51] [Tokenomics]
# Governance token distribution model

## NFT DAO Fund 3 Deliverable

**This Discussion was converted from an Issue to serve as a discussion thread for this project.**

**When assigning Issues related to this discussion :**

1. Add the Following Project Categories - [Governance token distribution model
](https://github.com/NFT-DAO/Governance-HOLON/projects/5) and [NFT DAO Fund 3 Deliverables](https://github.com/NFT-DAO/Governance-HOLON/projects/6).
1. Add the Following Milestone - [Sign-Off Code of Conduct](https://github.com/NFT-DAO/Governance-HOLON/milestone/6)

## Project Board
https://github.com/NFT-DAO/Governance-HOLON/projects/5

## GitHub Location 
TBA

## Discord Channel
TBA

## Triaging consensus levels

> "triaging consensus levels" what a brilliant term @tttroyyy ! 🧠 ⚡
> This gives me an idea for our "kudo" tokens: How to build a verified domain expertise from within the community!
> And also start to have a piece of the "tool" to automate consensus / voting

>    Each Holon is represented by two sets of tokens, one internal and one external
>   "Internal" tokens are eligible for voting on Holon decisions // decisions encompassing all Holons are voted on by all Holon internal tokens
>    "external" tokens: Collectible by individuals outside Holon ; given by anyone from within the Holon for adding value in any form to the Holon

> Once someone collected "enough" (?) external tokens from any given Holon, he can request to join the Holon by burning the external tokens and providing im with an initial supply of internal tokens.
> That way we can track and verify who has provided the most value in what area and can grow the Holons organically and more precise.

> Requirements

> The cost of minting all these tokens would be below 7 USD
>   Daedalus wallet for every participant
>  a dev with a Cardano CLI setup to mint the tokens, or wait for me to have it figured out 😆

![NFT-DAO-COmmunity-Governance-Pilot](https://user-images.githubusercontent.com/25156451/114104222-5f9af400-98c2-11eb-9071-2db4784db9fd.jpg)

**Comment by DanM3rcurius** [comment]
I'm waiting on input/feedback so I Can start minting tokens with Ancient Kraken. 
We need temporary names also for the tokens and decide how many holons we have/want to mint tokens for.
So far I have:
eGVNC / iGVNC 
eMRKT / iMRKT
eNGRN / iNGRN
ePRDT / iPRDT
eMVP / iMVP

@tttroyyy @hududed

**Comment by kopcho** [comment]
Hang on.... lots of unanswered questions here
________________________________

**Comment by hududed** [comment]
some comments, thoughts:
![image](https://user-images.githubusercontent.com/34112826/114288502-b1ac5880-9a2d-11eb-8f28-868b392bedc1.png)

Do we have our own native token for auction / marketplace transactions or are we using ADA only?
Are we entertaining ideas on liquidity pools? If yes, would it be NFTDAO-ADA, ND-ADA or N-ADA liquidity pools?
Would be great to tie such tokens to these reputation/contribution tokens - though I suspect this is not feasible.
As for token names, ones that include NFT-DAO would be ideal, but thats too long. What about:
Governance - NeG/NiG 
Marketing - NeM/NiM
Engineering - NeE/NiE
Product - NeP/NiP

**Comment by tttroyyy** [comment]
Hey Guys, sorry for the late reply. I've been off-line for most of the weekend. I'm drafting a response to this, which I will be able to tidy up tomorrow. There has been a lot of thought and consideration put into this flow-chart @DanM3rcurius and responses @hududed. As much as we have a goal to get minting quickly, there is also a lot of questions arising about 1) how the process of distributing our first token will map onto the current state of our community, 2) how our broader token distribution will be rolled out in the coming weeks/months and 3) the implications our immediate choices will have on the way the DAO functions in the near future. Far from a small task, but I feel excited that a resolution is nearing formation.

**Comment by DanM3rcurius** [comment]
👋 @hududed thanks for the highly appreciated feedback 🙏
Here are my answers/thoughts:
![IMG_9562](https://user-images.githubusercontent.com/80257662/114459385-c3822d00-9be0-11eb-910a-8f8ff4edc437.png)

1. yes each holon member 
2. Yes they get the same amount of internal tokens in each holon they are a member of (different tokens!)
3. All kudo tokens are inflationary, they belong to the utility token „sphere“. If they are depleted before the next iteration, then new ones with the same policy script will be minted. Sidenote: The governance token with voting rights (security token „sphere“)will be deflationary
4. Value added will be measured subjectively, comment with reference to action/value is necessary in token tx. The assessment criteria are a supreme idea! However to verify this manually will create a lot of overhead. Ideas? ALTERNATIVE: SourceCred!
5. The internal tokens (ND-i) worth is represented by holon voting power. we could increase incentive to acquire/hold & stake for minting of exclusive NFTs. Additionally it is like a reputation/contribution diary. They enable us to track: who contributed sth, how valuable (if multiple tokens are sent for same / similar event or action then apparently it had a greater impact) and when (for how long has she/he be contributing already). In the end this helps us to decide how to distribute the governance tokens, once we are ready to.
6. To keep it „simple“ for now I’d suggest to treat interholon kudos same as external community kudos. Meaning: giving kudos outside ones holon is always done via external token
7. The external tokens (ND-e) value is represented by community voting power. we could increase incentive to acquire/hold by random airdrops of NFT collectibles or similar. Additionally it is like a reputation/contribution diary. They enable us to track: who contributed sth, how valuable (if multiple tokens are sent for same / similar event or action then apparently it had a greater impact) and when (for how long has she/he be contributing already). In the end this helps us to decide how to distribute the governance tokens, once we are ready to.
8. Ada only for auction , afaik
9. Liquidity pools are not yet integrated into the tokenomy, how would you go about it? Why do we need them where, when and how?
10. I like the name proposals!

**Comment by DanM3rcurius** [comment]
> Hey Guys, sorry for the late reply. I've been off-line for most of the weekend. I'm drafting a response to this, which I will be able to tidy up tomorrow. There has been a lot of thought and consideration put into this flow-chart @DanM3rcurius and responses @hududed. As much as we have a goal to get minting quickly, there is also a lot of questions arising about 1) how the process of distributing our first token will map onto the current state of our community, 2) how our broader token distribution will be rolled out in the coming weeks/months and 3) the implications our immediate choices will have on the way the DAO functions in the near future. Far from a small task, but I feel excited that a resolution is nearing formation.


great points @tttroyyy !
1) We need to define who is in which holon in order to do a trial distribution. 
2) we need to collaborate on the overall tokenomics, I’ll start with a token engineering canvas tomorrow. Didn’t have time last days as I was occupied with other projects.
3) the value of starting now with an experiment outweighs the risk of not having any community reputation or kudo system, as community is the one thing a DAO needs. The earlier we integrate and the more we value our steadily growing community, the more likely it is we will succeed as a true DAO.

**Comment by tttroyyy** [comment]
[T. Egan - NFT-DAO Tokenomics Matrix (13-4-21).pdf](https://github.com/NFT-DAO/Governance-HOLON/files/6301921/T.Egan.-.NFT-DAO.Tokenomics.Matrix.13-4-21.pdf)

Sorry for the open and close (my bad). Here is what I have been thinking about with the tokenomics model. This outlay doesn't generate tokens according to holon-types as you've been exploring @DanM3rcurius. For this pre-alpha mint, we probably wouldn't need to go into that depth I think, but we can explore the concept for up-coming phases of the token rollout. I'll forward this for @kopcho, Tyler and Michael to look at too.
